tests/integration/targets/hcloud_network/tasks/main.yml

```diff
@@ -55,6 +55,31 @@
     that:
       - network is not changed
 
+- name: test create Network with expose_routes_to_vswitch
+  hcloud_network:
+    name: "{{hcloud_network_name_with_vswitch}}"
+    ip_range: "10.0.0.0/16"
+    expose_routes_to_vswitch: true
```

Comment:
Can you add a `state: absent` "cleanup" step at the beginning of this test similar to https://github.com/ansible-collections/hetzner.hcloud/pull/228?